### ISSUE TITLE
Ski reflectivity

### ISSUE THREAD

**Issue by legoboyvdlp** [c172p-team/c172p#1225]
Hi,
I'm not too sure about the reflections on these skis. 

There's quite a few strange things - is that to do with uvmapping?

![image](https://user-images.githubusercontent.com/10434911/47498717-fd66e980-d855-11e8-8f3d-449497598c0a.png)

**Comment by wlbragg** [comment]
Yeah, I was never really happy with the "chrome" surface reflections. I guess we need to take another look at it now with fresh eyes.
